README.md

```diff
@@ -8,7 +8,7 @@ beacon receiver for [boomerang].
 [![Build status][ci-image]][ci-status]
 
 * **boomcatch version**: *3.0.4*
-* **node.js versions**: *0.10, 0.12, latest stable io.js*
+* **node.js versions**: *0.10 - 4.0*
```

Comment:
This makes it look like you don't support 4.1, 4.2 or 5.0. I'd change it to `0.10 and later` to ditch the maintenance burden.